src/middleware/ChaosHandler.ts

```diff
@@ -141,10 +141,12 @@ export class ChaosHandler implements Middleware {
 	 */
 	private async sendRequest(chaosHandlerOptions: ChaosHandlerOptions, context: Context): Promise<void> {
 		this.setStatusCode(chaosHandlerOptions, context.request as string, context.options.method as RequestMethod);
-		if (!chaosHandlerOptions.statusCode) {
-			await this.nextMiddleware.execute(context);
-		} else {
+		if (Math.floor(Math.random() * 100) < chaosHandlerOptions.chaosPercentage) {
```

Comment:
This condition will always be checked even when the user selects 'ChaosStrategy.MANUAL'.  and  as ChaosPercentage defaults to 10 -> 
`this.chaosPercentage = chaosPercentage !== undefined ? chaosPercentage : 10;`

ChaosPercentage indicates the percentage of randomness/chaos in the handle. 

Setting the ChaosPercentage when using the manual strategy might be a confusing convention. 

I feel we should check for the chaos strategy as well and determine the condition.

Reply:
Makes sense. I'll do both!